include/sdsl/memory_management.hpp

```diff
@@ -746,12 +746,14 @@ class memory_manager {
 public:
 	static void use_hugepages(size_t bytes = 0)
 	{
-#ifndef MSVC_COMPILER
+#ifndef _WIN32   // MSVC_COMPILER
 		auto& m = the_manager();
 		hugepage_allocator::the_allocator().init(bytes);
 		m.hugepages = true;
 #else
-		throw std::runtime_error("hugepages not support on MSVC_COMPILER");
+		// avoid error: unused parameter 'bytes' [-Werror=unused-parameter]
+	    throw std::runtime_error(std::string("hugepages not support on _WIN32- not set bytes: ") +
+	                             std::to_string(bytes) );
```

Comment:
```suggestion
	    throw std::runtime_error(std::string("hugepages not supported on Windows"));
	    (void) bytes;
```
I'd just change the error message to say "Windows" and `(void)` the `bytes`. There's not really a point in printing `bytes`.